packages/pigment-css-react/README.md

```diff
@@ -663,6 +663,104 @@ declare module '@pigment-css/react/theme' {
 }
 ```
 
+## Right-to-left support
+
+Pigment CSS is designed for left-to-right (LTR) languages by default. To support right-to-left (RTL) languages, add the `dir="rtl"` attribute to your app's `<html>` element or any other equivalent top level container. Then, update your bundler config as follows:
```

Comment:
```suggestion
To support right-to-left (RTL) languages, add the `dir="rtl"` attribute to your app's `<html>` element or any other equivalent top level container. Then, update your bundler config as follows:
```

I would drop this. There is no design decision about this, one has to be default :)